docs/user_guide/01_install_medmodels.md

```diff
@@ -2,10 +2,14 @@
 
 ## Prerequisites
 
-## Local Installation
+Before installing MedModels, ensure you have the following installed on your system:
+
+- **Python:** Compatible with Python versions 3.10, 3.11, 3.12, or 3.13.
```

Comment:
```suggestion
Before installing MedModels, ensure your system has:

- **Python:** Version 3.10, 3.11, 3.12, or 3.13.
```

Reply:
a bit simpler. before it felt like a huge list with multiple entries would follow